e2e-test/cmd/e2e_helper/main.go

```diff
@@ -276,28 +278,61 @@ func (s *server) networkRecvTest(w http.ResponseWriter, r *http.Request) {
 }
 
 func (s *server) stressCondition(w http.ResponseWriter, r *http.Request) {
-	control, err := cgroups.Load(cgroups.V1, cgroups.PidPath(1))
-	if err != nil {
-		http.Error(w, err.Error(), http.StatusInternalServerError)
-		return
-	}
+	if cgroups.Mode() == cgroups.Unified {
+		group, err := cgroup2.PidGroupPath(1)
+		if err != nil {
+			http.Error(w, err.Error(), http.StatusInternalServerError)
 
-	stats, err := control.Stat(cgroups.IgnoreNotExist)
-	if err != nil {
-		http.Error(w, err.Error(), http.StatusInternalServerError)
-		return
-	}
+			return
+		}
 
-	response, err := json.Marshal(map[string]uint64{
-		"cpuTime":     stats.CPU.Usage.Total,
-		"memoryUsage": stats.Memory.Usage.Usage - stats.Memory.Kernel.Usage - stats.Memory.Cache,
-	})
-	if err != nil {
-		http.Error(w, "fail to marshal response", http.StatusInternalServerError)
-		return
-	}
+		m, err := cgroup2.Load(group)
+		if err != nil {
+			http.Error(w, err.Error(), http.StatusInternalServerError)
+
+			return
+		}
+
+		stats, err := m.Stat()
+		if err != nil {
+			http.Error(w, err.Error(), http.StatusInternalServerError)
+			return
+		}
+
+		response, err := json.Marshal(map[string]interface{}{
+			"cpuTime":     stats.CPU.UsageUsec * 1000,
+			"memoryUsage": stats.Memory.Usage - stats.Memory.KernelStack - stats.Memory.File,
```

Comment:
I guess Usage is enough.

Reply:
From @Andrewmatilde's advices, `stats.Memory.Usage` has already indicated the current memory usage. Here is the ref: https://github.com/containerd/cgroups/blob/main/cgroup2/manager.go#L625.